src/app/public/modules/repeater/repeater.component.html

```diff
@@ -1,8 +1,23 @@
-<div
-  class="sky-repeater"
-  dragula="sky-repeater"
-  [dragulaModel]="getItemsFromService()"
-  #repeaterHost
->
+<ng-container #repeaterHost>
+  <div *ngIf="!reorderable; else draggable"
+    class="sky-repeater"
+  >
+    <ng-container *ngTemplateOutlet="content">
+    </ng-container>
+  </div>
+</ng-container>
+
+<ng-template #draggable>
+  <div
+    class="sky-repeater"
+    [dragula]="dragulaGroupName"
```

Comment:
Couldn't we just set this to `undefined` instead of using the `ngIf else` ?

Reply:
I tried that, and even when set to `undefined` or even `null`, Dragula was still getting initialized.